### ISSUE TITLE
Grues cannot reasonably deal with wall mounted lights

### ISSUE THREAD

**Issue by Killette2** [d3athrow/vgstation13#33860]
![Screenshot_20221218_082449](https://user-images.githubusercontent.com/5169310/208349778-0e11bad6-9e45-418b-afc2-7fc869333a18.png)

With the insane DPS that light does to grues, the amount of wall lanterns makes it impossible to play. Was spawned in a corner near no vents between cargo and medbay and died before anything happened

**Comment by nervere** [comment]
@Killette2 What stage grue were you?

**Comment by Kromkar** [comment]
> @Killette2 What stage grue were you?

Grue larva (I looked at the screenshot and read his post)

**Comment by Eneocho** [comment]
Grue in general is very weak. An idea I read a while ago was to change the CONSOOM light from an toggle ability to an innate ability upon birth, improving with each stage. You could only consoom so much light before it started harming you. At larva stage you can tank a lighter's light indefinitely but get damaged with brighter lights. At teen you can tank PDAs or 1 wall mounted light indefinitely, but get damaged from multiple wall light sources. At adult you can tank 3 wall lights +1 per each consumed sentient.

**Comment by pedr69** [comment]
maybe lantern health and light intensity could also be lowered
it takes 3 hits to knock the wallmounted lantern down, and then another 3 to break the thing

**Comment by Hinaichigo** [comment]
Making lanterns a bit weaker could be reasonable.
And yeah maybe drain light could be somewhat reworked. Stage zero that you can get if you eat a monkey to become an adult isn't strong enough to negate a normal bulb, I wonder how people feel about that.
As a tactic versus the lanterns you can back off from attacking it after a few hits, to let the accumulative light sensitivity wear off so the next few ticks don't do as much damage. Now that I think of it that mechanic isn't so clearly explicated.
I like Eneocho's idea of having the brightness threshold to start getting damaged increase with life stage.

**Comment by xeroxemnas** [comment]
grues are fuckin trash and you cant even break half the lights that assistants use on you, if an assistant has a flare in his pocket or something you're boned. Attacking a person should automatically break or at least temporary disable all lights on their person, rewarding grues for repeat attacks and getting the drop on people.

**Comment by Eneocho** [comment]
To further the idea of innate light consumption increasing with stage, you could get an active to the innate ability that lets you consume more light than you should be able to tolerate.
Either a nutrient system (you can only consume while you have the 'energy' to do so) or a simple timer (you can consume more than you should normally for 15s).

**Comment by xeroxemnas** [comment]
yeah they definitely need some kind of buff. The idea of a grue is that it should make people AFRAID of going into dark areas, not just running in recklessly with a bunch of flares and owning the thing just by running around.

**Comment by Killette2** [comment]
@nervere I was actually a grue juvenile during that screenshot. Deadchat reverted me to larva kind of like with xenomorph antags and whatnot sometimes. The wiki says they have 150 health and and deal 15-20 damage and one wall mounted lantern still owned me

Proof: you can't force open doors til you're a juvy, and adults can do it instantly

**Comment by DorainGrey** [comment]
Nerfing wallmounted lanterns is good, some maps just have too much lighting in maint.

However any buffs need to bear in mind, Grues are a minor antag and shouldn't be creeped too far.
They aren't supposed to be powerful or able to stand up to a persistent crew hunting them, they thrive when the crew ignores them and allows them to power up to crazy levels. Similar to how catbeasts increase the threat.

**Comment by Hinaichigo** [comment]
I think one avenue to buff them would be to make them more evasive so they can't as easily be hunted down and slain, but not necessarily make them stronger. So there'd be a grue lurking around persistently that you just have to be cautious about and not go into the dark without a light. One option is a darkness blink that lets you warp from a dark tile to a random nearby dark area every so often or something like that, or further movespeed buffs.
Flares I deliberately didn't make breakable based on forum feedback but it'd be pretty simple to change that if people want. Probably the most glaring besides flares are floodlights and console screens no?

**Comment by pedr69** [comment]
way to make them more evasive might be a smokecloud spell
pop it in a maint intersection so the guy chasing you doesn't see which way you went

**Comment by Hinaichigo** [comment]
Along similar lines originally I was considering the idea of like an AOE darkness spell that sort of darkens a certain area for a while. It'd be rooted to that area rather than the grue itself like the current Drain Light.

Humanoid grues also had a shriek that shattered all the lights in a certain area around them.

**Comment by Hinaichigo** [comment]
I think this is actually a juvenile based on the attack text. The name just defaults to `grue larva (123)` for deadchat and I think that might have started happening after #33004.
So maybe the wall mounted lanterns are a bit overly durable.
